config/environments/test.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'logger/formatter/conjur_formatter'
+require 'test/audit_sink'
```

Comment:
How does it related to ruby 3?

Reply:
Failed to import this file during Integration tests.

Reply:
Double checked this one - keeping the new require